src/flow.c

```diff
@@ -1113,6 +1113,19 @@ void FlowUpdateState(Flow *f, enum FlowState s)
     }
 }
 
+/**
+ * \brief Get flow last time as individual values.
+ *
+ * Instead of returning a pointer to the timeval copy the timeval
+ * parts into output pointers to make it simpler to call from Rust
+ * over FFI using only basic data types.
+ */
+void FlowGetLastTimeAsParts(Flow *flow, uint64_t *secs, uint64_t *usecs)
+{
+    *secs = flow->lastts.tv_sec;
```

Comment:
would be it better to explicitly cast tv_sec and tv_usec to uint64_t here? Think depending on arch this can be signed types

Reply:
Good idea.